<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCustomFields</class>
 <widget class="QWidget" name="EditCustomFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="multiText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckList" name="checkList" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCheckList</class>
   <extends>QWidget</extends>
   <header>qchecklist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
